dev/docker-compose.yml

```
@@ -0,0 +1,7 @@
+version: '3'
+services:
+    naguis:
+        build:
+            context: .
```

Since the `Dockerfile` is one line, we can reference the image like so:

```suggestion
        image: jasonrivers/nagios
```